packages/create-gatsby/src/questions/cmses.json

```
@@ -1,12 +1,36 @@
 {
-  "gatsby-source-contentful": { "message": "Contentful" },
-  "gatsby-source-datocms": { "message": "DatoCMS" },
+  "gatsby-source-contentful": {
+    "message": "Contentful",
+    "dependencies": [
+      "gatsby-plugin-image",
+      "gatsby-plugin-sharp"
+    ]
+  },
+  "gatsby-source-datocms": {
+    "message": "DatoCMS",
+    "dependences": [
+      "gatsby-plugin-image",
+      "react-helmet"
```

```suggestion
      "gatsby-plugin-image"
```

react-helmet is not needed

---

Thanks!